src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainParameters.cs

```diff
@@ -9,6 +11,84 @@ namespace Nethermind.Specs.ChainSpecStyle;
 
 public class ChainParameters
 {
+
+    private static readonly Dictionary<string, Func<ChainParameters, ulong?>> HardforkToEips = new()
+    {
+        { "Dencun", cp => cp.Eip4844TransitionTimestamp },
+        { "Cancun", cp => cp.Eip4788TransitionTimestamp },
+        { "EIP-1153", cp => cp.Eip1153TransitionTimestamp },
+        { "EIP-5656", cp => cp.Eip5656TransitionTimestamp },
+        { "EIP-6780", cp => cp.Eip6780TransitionTimestamp }
+    };
```

Comment:
I was thinking of something more like:

```
    public static readonly IReadOnlyList<Expression<Func<ChainParameters, ulong?>>> DencunEIPs =
    [
        cp => cp.Eip4844TransitionTimestamp,
        cp => cp.Eip4788TransitionTimestamp,
        cp => cp.Eip1153TransitionTimestamp,
        cp => cp.Eip5656TransitionTimestamp,
        cp => cp.Eip6780TransitionTimestamp
    ];
```

If we had same interface for `ChainParameters` and `ChainSpecParamsJson` that could be potentially reusable.

But that maybe an overkill.